monitor/monitor.go

```diff
@@ -206,6 +210,28 @@ func New(opts MonitorOptions, stdout, stderr *log.Logger, clk clock.Clock) (*Mon
 	return m, nil
 }
 
+func makeDB(uri, passwordFile string) (storage.Storage, error) {
+	if passwordFile == "" {
+		return nil, fmt.Errorf("must provide a password file")
+	}
+	password, err := ioutil.ReadFile(passwordFile)
```

Comment:
How do you feel about also adding a check that the filesystem permissions on `passwordFile` are secure and erroring if they are too permissible? I really appreciate when tools (e.g. `ssh`) do this on my behalf. It often catches sloppy `umask` mistakes.